src/modules/user/user.controller.ts

```diff
@@ -1,9 +1,21 @@
-import { Controller, Patch, Param, Body, UsePipes, ValidationPipe, Request, Req } from '@nestjs/common';
+import {
+  Controller,
+  Patch,
+  Param,
+  Body,
+  UsePipes,
+  ValidationPipe,
+  Request,
+  Req,
+  Get,
+  HttpException,
+} from '@nestjs/common';
```

Comment:
Only import methods you are consuming. Removing all unnecessary methods from your import